src/Compilers/CSharp/Test/Emit3/Semantics/CollectionExpressionArgumentTests.cs

```diff
@@ -3997,6 +4101,1353 @@ static void Main()
                 Diagnostic(ErrorCode.ERR_DefaultLiteralNoTargetType, "default").WithLocation(7, 34));
         }
 
+        [Theory]
+        [CombinatorialData]
+        public void InterfaceTarget_ArrayInterfaces(
```

Comment:
nit: consider also testing `with(comparer)` and possibly `with(capacity, comparer)` in this test